api/LMPlatform.UI/Controllers/AccountController.cs

```diff
@@ -310,6 +317,57 @@ public string GenerateBase64(HttpPostedFileBase file)
             return "data:image/png;base64," + Convert.ToBase64String(thePictureAsBytes);
         }
 
+        [JwtAuth]
+        [HttpPost]
+        public ActionResult DeleteAccount()
```

Comment:
Lets start work with async/await approach. (and the same for all a new methods or actions)